### ISSUE TITLE
Workflow Build Error: Container Overrides length must be at most 8192

### ISSUE THREAD

**Issue by erikcw** [convox/rack#1906]
After upgrading to version `20170207013217` I've started getting the following error from my Workflow Jobs when they attempt to build:

```
***** TASK 1/2 [build] for job 8a5ae688-75b0-4857-8ff2-27d22e59c776

ERROR: InvalidParameterException: Container Overrides length must be at most 8192
	status code: 400, request id: bfe940ea-ee14-11e6-94e5-c72319bb644e
```

**Comment by erikcw** [comment]
I just updated the rack to `20170209003424` and the issue persists.

**Comment by mwarkentin** [comment]
Related: https://github.com/convox/rack/pull/1914

**Comment by mattmanning** [comment]
The fix for this was released in Rack version 20170210204814. Thanks for reporting.
